### ISSUE TITLE
Add version for KiCAD v7 and/or v8

### ISSUE THREAD

**Issue by DeflateAwning** [mfhepp/open_hardware_template#4]
It seems like this project is for v4? Can you please add a version for KiCAD v7 and/or v8? Thanks!

**Comment by mfhepp** [comment]
Thanks for the proposal; however, the project is not actively maintained and I do not have the resources to update it at the moment.

**Comment by DeflateAwning** [comment]
May I suggest archiving this repo if you are taking a strong stance against supporting it, or reopening this issue until it is completed (even if that is never). This project is still linked to, so it's helpful to see that this is an active reasonable request. You closed it as "completed", which it is not.

**Comment by mfhepp** [comment]
You are right - I will re-open this :-). But I will likely only update it once I will have completed my migration to V8, so there will be a V8 version, not a V7.

**Comment by DeflateAwning** [comment]
Thanks! I like that I created this a day before the v8 release; v8 is the way to go now.

**Comment by mfhepp** [comment]
I toyed around with Kicad 8 and looked into the current [Kicad templates mechanism](https://docs.kicad.org/8.0/en/kicad/kicad.html#creating_templates) and think that it will be better to turn this project into a proper KiCad template.

The proprietary `rename.sh` script would become obsolete by this, as far as I understand.

Note that all the additional folders and files (like `datasheets/` etc.) should be copied by Kicad automatically when using such an extended template. 

Notes to self:

1. It may be necessary to place "dummy" files into such placeholder directories. A `README.md` looks better to me than the historic `.gitkeep` technique.
2. Note that [Kicad will not copy files starting with a dot](https://docs.kicad.org/8.0/en/kicad/kicad.html#creating_templates), except for `.gitignore` and `.gitattributes`. Others may need to be handled manually: 
    > "Files with names beginning with the . character (dotfiles) are not copied. There is a special case for files named .gitignore or .gitattributes, which are copied if they exist."

**Comment by mfhepp** [comment]
**Todo:**

- [ ] Convert to V8
- [ ] Convert to Kicad Template
- [ ] Test if directories and additional files are copied properly
- [ ] Update documentation
- [ ] Add `meta/` folder with `info.html`
- [ ] Convert hero image to a 64 x 64 pixel PNG image and to `meta` as `icon.png`
